src/common.rs

```diff
@@ -410,6 +415,17 @@ pub(crate) fn log(text: &str) {
     }
 }
 
+// Utility function used to trigger a reboot of the operating system
```

Comment:
Yes, but _why_ does it flush the logger and sleep for 3 seconds? Why flush the logger at all? Why not sleep for 10 seconds? These are the questions a reader can't answer by inspection.

Reply:
tbh, this was simply moved from another section into common

the rationale as to why is not there and as with other parts of takeover, it's not documented and I can only hazard a guess